reserve_harvester.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+A script to gather Wikidata IDs of Swedish nature reserves.
+
+The input is a Petscan dump from svwp category "Naturreservat i Sverige",
+two levels deep (naturreservat -> nr i <län> -> nr i <kommun>):
+https://petscan.wmflabs.org/?psid=914993
+
+Then it's compared to NR_polygon.csv the following way:
+
+* The municipalities of each wp article are extracted
+* If a reserve in the csv file has exactly the same set of municipalities,
+as well as an identical or very similar name (Foo / Foo natturreservat),
+it's added to the exact matches file: svwp_to_nature_id_exact.json
+* If more than one match is found, they're added to
+svwp_to_nature_id_multiple.json, which can be reviewed manually.
+* If there's no match, the svwp article is added to
+svwp_to_nature_id_none, which can be reviewed manually.
```

Comment:
The script will be rerun, ideally the last time right before the actual upload, to get the freshest data. But of course there is value in keeping it in the repo, so that's it's easy to share and also to notice interesting parts in it.

I have added an empty "id" field to the output of "svwp_to_nature_id_none", so that it's easier to add an ID manually if possible.